python/mxnet/_numpy_op_doc.py

```diff
@@ -86,3 +86,50 @@ def _np_zeros_like(a):
         Array of zeros with the same shape and type as `a`.
     """
     pass
+
+
+def _npi_linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None, axis=0):
```

Comment:
This is not needed. We only need to put ops with prefixes `_np_` and `_npx_` here.

Reply:
done!